README.md

```diff
@@ -30,16 +31,13 @@ This repo contains 3 main folders:
 *Cohort 2 [Americas]: (started 2020-07-30) - Thursdays, 7:30pm EDT/EST*
 
 - 2020-08-20 Chapter 4 - Subsetting
-
 <details>
-  <summary> Week 2 Guiding Questions </summary>
+  <summary> Chapter 4 Guiding Questions </summary>
 
-  Week 2
-
-- What is the difference between an object’s name and its values?
-- When you modify a vector, when does a copy occur? Are there cases when a copy isn’t created?
-- How do you check how much space and object occupies? What kinds of objects occupy the most/least space?
-- When does R’s garbage collection usually run? How do you manually clean up memory?
+- Why are there six methods for subsetting atomic vectors? Are there different use cases or purposes for each? 
+- What are the similarities and differences between subsetting vectors, lists, dataframes, and tibbles? 
+- What is the difference between simplifying and preserving when subsetting? 
+- What are the most common use cases for subsetting? 
```

Comment:
I like the idea of making a guiding questions its own markdown OR I can make a div 

```
:::guidingq
:::
```

So we start each chapter of the bookdown with these questions? I kind of love that idea.... Thoughts?

Reply:
I like this, I know Jon was working on similar. Wonder where to fit it into the READMEs.

Reply:
I like this idea as well - starting each chapter with guiding questions.